hw5-2/main.c

```diff
@@ -0,0 +1,57 @@
+#include <stdio.h>
+#include "stack.h"
+
+int isBracketsConnected(char fstChar, char sndChar) {
+    return ((fstChar == '(') && (sndChar == ')')) ||
+           ((fstChar == '{') && (sndChar == '}')) ||
+           ((fstChar == '<') && (sndChar == '>')) ||
+           ((fstChar == '[') && (sndChar == ']'));
+}
+
+int isClosedBracket(char stringChar) {
```

Comment:
```suggestion
bool isClosedBracket(char stringChar) {
```
Функция отвечает да или нет, столо быть возвращать надо bool.

Reply:
Это верно и для остальных функций.